### PR TITLE
YLP-2088 build header component for historized settings table

### PR DESCRIPTION
**Pull request by Nas84**
This PR is a split of a biggest one #1064  in order to facilitate review, i propose to maintain the checklist on the main one only. what do you think ? 

## Checklist
- [ ] PR name contains the Jira ticket number
- [ ] Changelog updated
- [ ] Code tested (unit tests and integration tests)
- [ ] No console errors/warnings in integration tests
- [ ] No translation files updated (except for English files)
- [ ] Code formatted on every file
- [ ] Copyright added on every file
- [ ] End-to-end tests successful on CI

### CODE REVIEW

File: packages/dumb/src/components/settings/history/history-table-header.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ */
+
+import React, { FunctionComponent } from 'react'
+import styles from '../diabeloop.css'
+import { useTranslation } from 'react-i18next'
+
+export const HistoryTableHeader: FunctionComponent = () => {
+  const { t } = useTranslation('main')
+  return (
+    <thead key={'thead_4'}>
```
Comment (TimCanu) [inline packages/dumb/src/components/settings/history/history-table-header.tsx:35]: **🤓 nitpick:** attribute `key` is no longer necessary (same for all `key` used in this file)‏
```suggestion
    <thead>
```

File: packages/dumb/src/components/settings/history/history-table-header.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      <tr>
+        <th key="level" className={styles.secondaryLabelAlone}>
```
Comment (TimCanu) [inline packages/dumb/src/components/settings/history/history-table-header.tsx:37]: **❔ question:** ‏`secondaryLabelAlone` is not defined in `packages/dumb/src/components/settings/diabeloop.css`. Either your style is unecessary, or it needs to be added, which one is it? :slightly_smiling_face: ‏

Comment (Nas84) [reply packages/dumb/src/components/settings/history/history-table-header.tsx:37]: good catch 👍🏽

File: packages/dumb/src/components/settings/history/history-table-header.tsx (hunk 1 above)
Comment (Nas84) [reply packages/dumb/src/components/settings/history/history-table-header.tsx:35]: done

**Review by TimCanu — APPROVED** [review]

**Pull request merged**